share/smack/include/pthread.h

```diff
@@ -0,0 +1,71 @@
+//
```

Comment:
We should go over these key files in person once you are back. And also we should merge before we go over them.